store/sqlstore/upgrade.go

```diff
@@ -818,6 +818,7 @@ func upgradeDatabaseToVersion526(sqlStore SqlStore) {
 	// TODO: uncomment when the time arrive to upgrade the DB for 5.26
 	//if shouldPerformUpgrade(sqlStore, VERSION_5_25_0, VERSION_5_26_0) {
 	sqlStore.CreateColumnIfNotExists("Sessions", "ExpiredNotify", "boolean", "boolean", "0")
+	sqlStore.CreateColumnIfNotExistsNoDefault("Commands", "PluginId", "VARCHAR(190)", "VARCHAR(190)")
```

Comment:
We seem to use `CreateColumnIfNotExistsNoDefault` relatively sparingly -- can you expand on the consequences for not defining a default in this case? Does the field default to `NULL` or `""`? Does the migration on an existing database cause any weirdness between MySQL or Postgres?

Reply:
Originally I had it as `CreateColumnIfNotExistsNoDefault`, and then changed it to `CreateColumnIfNotExists`, but then the circleCI tests started failing. I'll try again, and hope it doesn't.
If it does, I think a NULL would still turn into a Go zero value in the struct, wouldn't it?

Reply:
@lieut-data It looks like it's failing because of this. Is it okay if we go with no default? (Unless you can think of a fix for the schema test?)

Reply:
Ok, we definitely /cannot/ leave it with NoDefault. It will break because somewhere along the way the sql (driver?) is returning it cannot convert NULL to string. So I need to figure out how to make the default work, but I'm just not sure why the shema test is failing.

Reply:
Discussed offline, will be going with `CreateColumnIfNotExistsNoDefault` because it matches what Gorp gives us.